sys/net/network_layer/ng_ipv6/ng_ipv6.c

```diff
@@ -66,6 +67,11 @@ kernel_pid_t ng_ipv6_init(void)
     return _pid;
 }
 
+kernel_pid_t ng_ipv6_getpid(void)
+{
+    return _pid;
+}
```

Comment:
(sidenote: why is Java always cited as a negative example for good capsulation. I know its better to have it more implicit than explicit in embedded systems, but its a common concept in most well-engineered systems ;-))